docs/core-features/table_lookups.md

```diff
@@ -101,8 +104,7 @@ assert np.array_equal(actual_output, expected_output)
 
 ### Transparent TLU
 
-For lot of programs, users will not even need to define their table lookup, it will be done
-automatically by Concrete.
+In many cases, you won't need to define your own TLUs, as Concrete can automatically perform these lookups.
```

Comment:
I would replace "as Concrete can automatically perform these lookups." by "as Concrete will set them for you". In your sentence, you've changed a bit the meaning